spectator-agent/src/main/resources/bitbucket.conf

```diff
@@ -0,0 +1,132 @@
+// http://confluence.atlassian.com/bitbucketserver/enabling-jmx-counters-for-performance-monitoring-776640189.html
+netflix.spectator.agent.jmx {
+  mappings = ${?netflix.spectator.agent.jmx.mappings} [
+    {
+      query = "com.atlassian.bitbucket.thread-pools:name=*"
+      measurements = [
+        {
+          name = "bitbucket.{name}.MaximumPoolSize"
```

Comment:
What is `name` in this example? I'm assuming the name of the thread pool which could be dynamic, in which case it should probably get moved to be a tag. The name and tag keys on a metric id should be deterministic and not contain any dynamic elements. The only dynamic part should be the values for some tags.